spec/factories/reports/realtime/bad_routing.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :bad_routing, class: Report::Realtime::BadRouting do
```

Comment:
```suggestion
  factory :bad_routing, class: Report::Realtime::BadRouting, parent: :cdr do
```

and remove copied trait :with_id_and_uuid